src/js/view/controls/controlbar.js

```diff
@@ -37,6 +38,21 @@ function div(classes) {
     return element;
 }
 
+function isRecsUpDisplay(abConfig) {
+    const tests = abConfig ? abConfig.tests : null;
```

Comment:
Returning early would simplify the logic of checking if `abConfig.tests.discoverIcon` is truthy.